test/unit_test/passes/onnx/test_perf_tuning.py

```diff
@@ -5,26 +5,33 @@
 import logging
 import math
 import re
-from test.unit_test.utils import create_dataloader, get_onnx_model
+from test.unit_test.utils import get_onnx_model
 from unittest.mock import MagicMock, PropertyMock, patch
 
 import psutil
 import pytest
+from torch.utils.data import DataLoader
 
+from olive.data.component.dataset import DummyDataset
 from olive.evaluator.metric_result import flatten_metric_result
 from olive.evaluator.olive_evaluator import OliveEvaluator, OnnxEvaluator
 from olive.hardware.accelerator import DEFAULT_CPU_ACCELERATOR, DEFAULT_GPU_CUDA_ACCELERATOR, AcceleratorSpec, Device
 from olive.passes.olive_pass import create_pass_from_dict
 from olive.passes.onnx.perf_tuning import PERFTUNING_BASELINE, OrtPerfTuning, PerfTuningRunner, generate_test_name
 
 
+# TODO(shaahji): Remove this once perf_tuning pass supports DataConfig
+def _create_dummy_dataloader(data_dir, batch_size=1, **kwargs):
+    return DataLoader(DummyDataset([[1, 1]]), batch_size=batch_size)
+
+
 @pytest.mark.parametrize(
     "config",
     [
         {"input_names": ["input"], "input_shapes": [[1, 1]]},
         {},
-        {"dataloader_func": create_dataloader},
-        {"dataloader_func": create_dataloader, "dataloader_func_kwargs": {"dummy_kwarg": 1}},
+        {"dataloader_func": _create_dummy_dataloader},
```

Comment:
can this use the `create_dummy_dataloader` from `test.unit_tests.utils`?

Reply:
Done!